draft-ietf-quic-recovery.md

```diff
@@ -1557,7 +1557,7 @@ OnLossDetectionTimeout():
     SetLossDetectionTimer()
     return
 
-  if (bytes_in_flight > 0):
+  if (any ack-eliciting packets in flight):
```

Comment:
I wonder if it is worth inverting this check here, so that the condition matches the above one.  It's a negative, but the any/no disparity is odd.  The other call site in SetLossDetectionTimer uses "no ack-eliciting packets in flight" as well.

Reply:
Good thought; applied.